Library/ViewModels/ThanksViewModel.swift

```diff
@@ -86,11 +86,25 @@ public final class ThanksViewModel: ThanksViewModelType, ThanksViewModelInputs,
 
     self.backedProjectText = self.configureWithDataProperty.signal
       .skipNil()
-      .map { project, _, _, _ in
-
-        let string = Strings.You_have_successfully_backed_project_html(
-          project_name: project.name
-        )
+      .map { project, _, _, pledgeTotal in
+
+        let string: String
+
+        if featurePostCampaignPledgeEnabled(),
+          project.isInPostCampaignPledgingPhase,
+          let email = AppEnvironment.current.currentUserEmail {
```

Comment:
This dependency on `currentUserEmail` is a little bit awkward - but it also seemed like a shame to add another fetch to what is currently a static screen, especially when `currentUserEmail` already was supposed to be there...